testsuite/integration-arquillian/pom.xml

```diff
@@ -320,6 +320,157 @@
+                                <execution>
+                                    <id>start-container</id>
+                                    <phase>process-test-classes</phase>
+                                    <goals>
+                                        <goal>start</goal>
+                                    </goals>
+                                    <configuration>
+                                        <showLogs>true</showLogs>
+                                        <images>
+                                            <image>
+                                                <alias>testdb</alias>
+                                                <name>${docker.database.image}</name>
+                                                <run>
+                                                    <ports>
+                                                        <port>3306</port>
+                                                    </ports>
+                                                    <env>
+                                                        <MYSQL_DATABASE>${keycloak.connectionsJpa.database}</MYSQL_DATABASE>
+                                                        <MYSQL_USER>${keycloak.connectionsJpa.user}</MYSQL_USER>
+                                                        <MYSQL_PASSWORD>${keycloak.connectionsJpa.password}</MYSQL_PASSWORD>
+                                                        <MYSQL_RANDOM_ROOT_PASSWORD>true</MYSQL_RANDOM_ROOT_PASSWORD>
+                                                    </env>
+                                                    <wait>
+                                                        <tcp>
+                                                            <ports>
+                                                                <port>3306</port>
+                                                            </ports>
+                                                        </tcp>
+                                                        <time>30000</time>
+                                                    </wait>
+                                                </run>
+                                            </image>
+                                        </images>
+                                    </configuration>
+                                </execution>
+                                <execution>
+                                    <id>stop-container</id>
+                                    <phase>post-integration-test</phase>
+                                    <goals>
+                                        <goal>stop</goal>
+                                    </goals>
+                                </execution>
+                            </executions>
+                        </plugin>
+                    </plugins>
+                </pluginManagement>
+            </build>
+        </profile>
+        <profile>
```

Comment:
Profiles for `oracle` and `mssql` are needed too (it should only be matter of setting the properties)

Reply:
If this PR would be ready in terms that the definition of a new database would be matter of creating a profile which would only consist of properties definition (`docker.database.image` alike) and adding env variables to docker run part, then yes, it can be done as a follow-up task.